src/completed/completed.service.ts

```
@@ -7,96 +10,111 @@ export class CompletedService {
     private prisma: PrismaService
   ){}
 
-  async todayTodosGet(user_id: number) {
+  // 오늘 TODO 목록 가져오기
+  // 타임존 문제가 있어서 시간이 이상하다. 한국 기준으로 하고 싶은데, 이것을 한국 기준으로 수정해서 하는것이 옳은지 모르겠다.
+  async todayTodosGet(userId: number): Promise<SuccessResponse | FailResponse | ErrorResponse> {
```

ESLint

[error] Replace `userId:·number` with `⏎····userId:·number,⏎··`

(prettier/prettier)